src/components/login.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import { AuthorizationContext } from "../authorizationProvider";
+import { Form } from "react-final-form";
+import FormTextField from "../components/common/formTextField"
+
+class Login extends Component {
+
+  onSubmit = async values => {
+    await this.context.login(values);
+  }
+
+  render() {
+    return (
+      <Form
```

Comment:
Po zmene na final-form tu chyba validacia :) (to by trebalo vo vsetkych formularoch)

Reply:
samozrejme.. TO DO